lib/freddy.rb

```
@@ -111,6 +111,15 @@ def respond_to(destination, &callback)
   # @option options [String] :group
   #   only one of the listeners in given group will receive a message. All
   #   listeners will receive a message if the group is not specified.
+  # @option options [Boolean] :durable
+  #   Should the consumer queue be durable? Default is `false`. This option is
```

Can we raise an error instead of ignoring?

---

The error is always re-raised

---

Ah, you mean if option `durable: true` is combined with `group: false`? Sure, can do